src/main/java/org/example/internetshop/controller/order/CompleteOrderController.java

```diff
@@ -24,7 +24,9 @@
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute("user_id");
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
         List<Product> products = shoppingCartService.getAllProducts(shoppingCart);
```

Comment:
```suggestion
        List<Product> products = shoppingCart.getProducts();
```
тобі тут не потрібно викликати ще раз сервіс, так як у нас вже є shoppingCart звідки можна витягнути продукти